src/components/Stepper.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+export default function Stepper({ isactive }) {
```

Comment:
Why the name `Stepper`?

Reply:
its common notation to call it stepper as it defines the 3 steps that the user will go through